### PR TITLE
Add python3 support

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@nthomas-redhat , @cloudbehl , @anmolsachan Please review

**Comment by TimothyAsirJeyasing** [comment]
@GowthamShanmugam , @shtripat , Please review

**Comment by TimothyAsirJeyasing** [comment]
@GowthamShanmugam , @cloudbehl , @anmolbabu , Please review

**Comment by TimothyAsirJeyasing** [comment]
@shtripat Please review

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import subprocess
 try:
     # Python 2 backwards compat
-    from __builtin__ import raw_input as input
+    from builtins import raw_input as input
```
Comment (aruniiird) [inline setup.py]: as @GowthamShanmugam suggested
```suggestion
    from builtins import input
```

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 2]
```diff
@@ -923,10 +923,7 @@ def get_device_tree():
         out, err, rc = cmd.run()
         if not err:
             out = out.encode('utf8')
-            devlist = map(
-                lambda line: dict(zip(keys, line.split(' '))),
-                out.splitlines()
-            )
+            devlist = [dict(list(zip(keys, line.split(' ')))) for line in out.splitlines()]
```
Comment (aruniiird) [inline tendrl/gluster_integration/sds_sync/__init__.py]: again, if you could add a comment, that would be better...

**Review by aruniiird — CHANGES_REQUESTED** [review]
Just some minor requests...

File: setup.py (hunk 1 above)
Comment (TimothyAsirJeyasing) [reply setup.py]: Done

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 2 above)
Comment (TimothyAsirJeyasing) [reply tendrl/gluster_integration/sds_sync/__init__.py]: Sure, thank you!

**Review by aruniiird — COMMENTED** [review]
looks good to me

**Review by aruniiird — APPROVED** [review]

**Pull request merged**